styles/home.css

```diff
@@ -146,3 +155,169 @@ footer {
 .sns-link img {
   width: 18px;
 }
+
+
+@media (max-width: 1218.75px) {
```

Comment:
그렇군요. 스크롤이 생겨서 화면이 밀리길래 (스크롤 두께 18.75px) 추가한 부분인데, 
content max-width를 1200px로 하고 그전까지 width를 100%로 하는 게 더 적절해 보여서 변경했습니다. 
잠시 임시방편 해결법을 push했다가 이 방법으로 다시 force push 했는데 문제는 없겠죠..?
[design(home-content): Solve scroll problem using max-width instead of…](https://github.com/codeit-bootcamp-frontend/6-Sprint-Mission/pull/160/commits/ca01654ebfcd9aaa990e5358d84792234c05c92d) [ca01654](https://github.com/codeit-bootcamp-frontend/6-Sprint-Mission/pull/160/commits/ca01654ebfcd9aaa990e5358d84792234c05c92d)

Reply:
괜찮습니다 ㅎㅎ

Reply:
git add 안한채로 git commit --amend 했어서 새 브랜치에 다시 적용했습니다.